src/Cards.svelte

```diff
@@ -51,7 +51,9 @@
     {/each}
 
     {#each profiles as profile}
-      <Card {profile} {config} path={`/${profile.nameOrAddress}`} />
+      {#if profile}
+        <Card {profile} {config} path={`/${profile.nameOrAddress}`} />
+      {/if}
     {/each}
```

Comment:
This API doesn't make sense anymore: why would a developer pass a list of `null`s instead of profiles? It won't render anything :)

Reply:
Done in [fc8360d](https://github.com/radicle-dev/radicle-interface/pull/327/commits/fc8360dd71d450efb2ed7e46aa9635e6fa465769)

Reply:
Yeah went back with all the `profile could be null` checks